src/db.rs

```diff
@@ -784,6 +784,19 @@ impl Builder {
             Err(StorageError::Io(io::Error::from(ErrorKind::InvalidData)).into())
         }
     }
+
+    /// Open an existing or create a new database in the given `file`.
+    ///
+    /// The file must be empty or contain a valid database.
+    pub fn build(&self, file: File) -> Result<Database, DatabaseError> {
```

Comment:
I think I like `open_file` better. Maybe if I ever do a 2.0 release, I'll change `open` to `open_path`.

Also, I think this needs the same empty file test that `open` has

Reply:
> Also, I think this needs the same empty file test that open has

Since the implementation is based on that of `create`, I was under the impression that a newly created empty file would be initialized with a new database?

Reply:
Ah right, I see. In that case lets name it `create_file` -- open to other suggestions on the name, but I think indicating that it's the same functionality as `create` is good.

Reply:
Changed it to `create_file` and added a test case for the creation of new database in an empty (temporary) file.